lib/index.js

```diff
@@ -0,0 +1,27 @@
+var _ = require('lodash');
```

Comment:
I've called it `index` because it's the only file now and I don't know what others will be called.
Since it's a module it does not make much difference and in future we could change it.